src/components/PodcastList/index.module.scss

```diff
@@ -0,0 +1,38 @@
+.root {
+  :global(.slick-list) {
+    overflow: visible;
+  }
+}
+
+.sectionTitle {
+  padding-top: 84px;
+  padding-bottom: 28px;
+}
+
+.item {
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 84px;
```

Comment:
This bottom padding is clickable, which means if I click below the card, it will take me to its link.
The worst case is when I try to click the arrows and miss the click in the arrow. It will take me to the podcast page above the arrows.

<img width="659" alt="Screenshot 2023-07-11 at 14 22 26" src="https://github.com/subvisual/subvisual.com/assets/36553777/7b6e3c83-005d-4ada-a329-7a20dd0127ec">